src/Content/Nav.php

```diff
@@ -211,11 +210,11 @@ private static function getInfo(App $a): array
 			$homelink = DI::session()->get('visitor_home', '');
 		}
 
-		if ((DI::args()->getModuleName() != 'home') && (! (Session::getLocalUser()))) {
+		if ((DI::args()->getModuleName() != 'home') && (! (DI::userSession()->getLocalUserId()))) {
```

Comment:
Too many parentheses!
```suggestion
		if (DI::args()->getModuleName() != 'home' && !DI::userSession()->getLocalUserId()) {
```